pokemongo_bot/cell_workers/pokemon_catch_worker.py

```diff
@@ -380,7 +380,7 @@ def work(self, response_dict=None):
                                                 data={'pokemon': pokemon_name}
                                             )
                             break
-        time.sleep(5)
+        time.sleep(2)
```

Comment:
I also wonder the impact of this. It was originally 5 because that is how long the app takes. Changing it to two would make it faster, but would also not match what is possible. Perhaps making this configurable would also have a large enough impact on the tick loop that we wouldn't need to have internal loops in the tasks that break our model of how `tick` works.